unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -448,6 +457,41 @@ public void SetAgentMode(string mode)
                 // limit camera from looking too far down
                 this.horizonAngles = new float[] { 30.0f, 0.0f, 330.0f };
             }
+
+            else if(whichMode == "drone")
+            {
+                //toggle first person character cull
+                fpcc.SwitchRenderersToHide(whichMode);
+
+                VisibilityCapsule = DroneVisCap;
+                m_CharacterController.center = new Vector3(0,0,0);
+                m_CharacterController.radius = 0.2f;
+                m_CharacterController.height = 0.0f;
+
+                CapsuleCollider cc = this.GetComponent<CapsuleCollider>();
+                cc.center = m_CharacterController.center;
+                cc.radius = m_CharacterController.radius;
+                cc.height = m_CharacterController.height;
+
+                m_Camera.GetComponent<PostProcessVolume>().enabled = false;
+                m_Camera.GetComponent<PostProcessLayer>().enabled = false;
+
+                //camera position set forward a bit for drone
+                m_Camera.transform.localPosition = new Vector3(0, 0, 0.2f);
```

Comment:
same comment as above

Reply:
same as above as well, although maybe there is a way to store these values on a drone controller script like you suggested that inherits from `PhysicsRemoteFPSAgent` but I think there will be some issues with execution order... will have to double check that.